x/stake/client/cli/flags.go

```diff
@@ -8,14 +8,16 @@ import (
 
 // nolint
 const (
-	FlagAddressDelegator    = "address-delegator"
-	FlagAddressValidator    = "validator"
-	FlagAddressValidatorSrc = "addr-validator-source"
-	FlagAddressValidatorDst = "addr-validator-dest"
-	FlagPubKey              = "pubkey"
-	FlagAmount              = "amount"
-	FlagSharesAmount        = "shares-amount"
-	FlagSharesPercent       = "shares-percent"
+	FlagAddressDelegator           = "address-delegator"
+	FlagAddressValidator           = "validator"
+	FlagAddressValidatorSrc        = "addr-validator-source"
+	FlagAddressValidatorDst        = "addr-validator-dest"
+	FlagAddressSmartChainOperator  = "address-smart-chain-operator"
```

Comment:
How about `address-smart-chain-validator`? Operator is too general here in my feelings.

Reply:
Done